resources/qml/WelcomePages/AddPrinterByIpContent.qml

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) 2019 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+
+import UM 1.3 as UM
+import Cura 1.1 as Cura
+
+
+//
+// This component contains the content for the 'by IP' page of the "Add New Printer" flow of the on-boarding process.
+//
+Item
+{
+    UM.I18nCatalog { id: catalog; name: "cura" }
+
+    id: addPrinterByIpScreen
+
+    property bool hasSentRequest: false
```

Comment:
Some documentation; What do these types mean? When will it be true?

Reply:
Added docs in a8e2fcf5f